app/controllers/api/repos_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Api::ReposController < ApplicationController
+    before_action :get_orgs
+    def index
+      render json: { repositories: @organization.repositories}, status: :ok
+    end
+
+    private
+    def get_orgs
+        @organization = @user.organizations.where(name: params[:org_id]).first

```

Comment:
If @user.organizations is null will it still work?

Reply:
o lmao oop didnt test that will have to fix